cmd/podman/push.go

```diff
@@ -135,23 +134,22 @@ func pushCmd(c *cli.Context) error {
 		}
 	}
 
-	options := libpod.CopyOptions{
-		Compression:         archive.Uncompressed,
-		SignaturePolicyPath: c.String("signature-policy"),
-		DockerRegistryOptions: common.DockerRegistryOptions{
-			DockerRegistryCreds:         registryCreds,
-			DockerCertPath:              certPath,
-			DockerInsecureSkipTLSVerify: skipVerify,
-		},
-		SigningOptions: common.SigningOptions{
-			RemoveSignatures: removeSignatures,
-			SignBy:           signBy,
-		},
-		AuthFile:         c.String("authfile"),
-		Writer:           writer,
-		ManifestMIMEType: manifestType,
-		ForceCompress:    c.Bool("compress"),
+	dr := image.DockerRegistryOptions{
```

Comment:
Could you spell this out a little better then dr.  registryOptions :=

Reply:
fixed